docker/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN npm install
 RUN mkdir frontend
 WORKDIR frontend
 COPY frontend/package.json .
-COPY frontend/yarn.lock .
+COPY --chown=pontoon:pontoon frontend/yarn.lock .
```

Comment:
I have no idea why, but docker stopped building without this. 

Side-note: I'm starting to hate create-react-app more and more…